net/libreswan/patches/010-nanosleep.patch

```diff
@@ -9,7 +9,7 @@
  #include <errno.h>
  
  #include "defs.h"
-@@ -192,7 +192,8 @@ static bool send_shunks(const char *wher
+@@ -200,7 +200,8 @@ static bool send_shunks(const char *wher
```

Comment:
This patch was primary for uClibc-ng (https://github.com/openwrt/packages/commit/c1a9e69feff5c879ddcdf476f59285b137c05ecd). Was is discussed in upstream in the past? 😇  It would be really great, but since uClibc-ng  was removed in 2021 while looking here: https://git.openwrt.org/?p=openwrt/openwrt.git;a=commitdiff;h=6b2ed6101e4c97691ba3aa94f25895be536b3870

Why do we still need to carry this patch?